src/apps/replication/meta_server/meta_service.cpp

```diff
@@ -81,23 +95,17 @@ error_code meta_service::start()
 
     // recover server state
     while ((err = _state->on_become_leader()) != ERR_OK)
-    {
-        derror("recover server state failed, err = %s, retry ...");
-    }
-    ddebug("recover server state succeed");
+        derror("recover server state failed, err = %s, retry ...", err.to_string());
```

Comment:
let's have { and } even there is only one stmt as a convention, which is helpful when the macros used may expand the stmt into multiple stmts.